### ISSUE TITLE
(1.12.2) Crash When Using Blood Lamp Sigil

### ISSUE THREAD

**Issue by TechnoMysterio** [WayofTime/BloodMagic#1243]
Spawned in Blood Lamp Sigil, tried to use it on the ground, game crashed. I didn't use any blood orbs beforehand, so I think it tried to look for blood in my network, couldn't find a network, and then crashed as a result. Pastebin here:

https://pastebin.com/WDVCiik4

- BloodMagic: 1.12.2-2.2.7-90
- Minecraft: 1.12.2
- Forge: 14.23.2.2624

**Comment by TehNut** [comment]
*angery pepe*

**Comment by zzBuilderzz** [comment]
you thought using an item from blood magic wouldn't cost blood? what planet do you come from?

**Comment by TechnoMysterio** [comment]
I come from the planet 'Loopy', which orbits around the star 'Mumbling To Yourself In Public', which can be found near the edge of the galaxy 'Two Sandwiches Short of a Picnic'.

**Comment by zzBuilderzz** [comment]
lol

**Comment by querns** [comment]
Related to this crash: I received a similar crash while using a Sigil of the Fast Miner.

Crash report: https://pastebin.com/v5eqLWPL

After doing some testing, I was able to pin down the source of the problem. I created a Sigil of the Fast Miner, and I forgot to bind it to myself by right clicking. I then activated the sigil by shift-right-clicking, and got the crash above. Loading my world again would continue to generate the same crash. After using NBTExplorer to delete the item from my inventory, I gave myself a new Sigil of the Fast Miner, carefully bound it to myself using the normal, in-game method, then activated it, and everything worked fine.
